docs/source/diagnostics-distributed.rst

```diff
@@ -124,12 +124,45 @@ The ``progress`` function takes a Dask object that is executing in the backgroun
    x.compute()      # convert to final result when done if desired
 
 
-External Documentation
-----------------------
+Connecting to the Dashboard
+---------------------------
 
-More in-depth technical documentation about Dask's distributed scheduler is
-available at https://distributed.dask.org/en/latest
+Some computer networks may restrict access o certain ports or only allow access
```

Comment:
Some computer networks may restrict access to certain ports or only allow access